examples/data-solutions/dp-foundation/01-lnd-main.tf.tf

```diff
@@ -0,0 +1,90 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# tfdoc:file:description Landing project
+
```

Comment:
I would rename all `nn-sss-main.tf` as `nn-sss.tf` and combine resources from `-main` and `-vpc`. I would also use more descriptive names.

This for example should become `01-landing.tf` and combine project and VPC resources.

Reply:
Done, update also other files.